dist/src/main/resources/application.properties

```diff
@@ -52,4 +52,8 @@ spring.autoconfigure.exclude=\
   org.springframework.boot.autoconfigure.elasticsearch.ElasticsearchRestClientAutoConfiguration, \
   org.springframework.boot.autoconfigure.ldap.LdapAutoConfiguration, \
   graphql.kickstart.autoconfigure.annotations.GraphQLAnnotationsAutoConfiguration, \
-  graphql.kickstart.autoconfigure.tools.GraphQLJavaToolsAutoConfiguration
+  graphql.kickstart.autoconfigure.tools.GraphQLJavaToolsAutoConfiguration, \
+  org.springframework.boot.autoconfigure.jdbc.DataSourceAutoConfiguration, \
+  org.springframework.boot.autoconfigure.jdbc.DataSourceAutoConfiguration, \
+  org.springframework.boot.autoconfigure.orm.jpa.HibernateJpaAutoConfiguration, \
+  org.springframework.boot.autoconfigure.jdbc.DataSourceTransactionManagerAutoConfiguration
```

Comment:
@Ben-Sheppard, with adding additional spring dependencies, Spring does a lot of auto-configuration stuff. I excluded these once and imported them in `IdentityModuleConfiguration` so that they are only included when the identity module starts.

Could you please ensure (double-check) that Spring only auto-configures what is needed for Identity?

Reply:
Hey @romansmirnov - I've looked here and I believe you have added all that is required for Identity, at least from my perspective, nothing else should be autoconfigured. I've also tested the app locally and it functions still.